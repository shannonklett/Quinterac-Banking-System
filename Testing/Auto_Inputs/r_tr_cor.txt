login
retail
transfer
000001
000002
100000
logout