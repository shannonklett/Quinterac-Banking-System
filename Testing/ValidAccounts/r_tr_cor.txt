000001
000002
000000
