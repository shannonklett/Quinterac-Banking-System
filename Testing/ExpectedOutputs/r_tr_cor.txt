03 000001 000002 00100000                
00 000000 000000 00000000                
